### ISSUE TITLE
Working in a pre-authenticated environement

### ISSUE THREAD

**Issue by JRGranell** [swagger-api/swagger-ui#3183]
**TL;DR**

The user has already authenticated - and I would like to pass the previously acquired JWT to swagger-ui 3.x to use for the security definitions and schemas.

Hi, 

Great library - just struggling to get this working when the user has already authenticated as neither our documentation nor API are publicly accessible. The user logs in and then I want to use the OAuth access token that is already available to call the API. As such swagger-ui is wrapped inside another web-app that handles authentication to achieve this.

Although I haven't used Swagger 2.x I understand that an 'authorzations' object can be passed in when creating a UI bundle as detailed in the swager-js migration guide https://github.com/swagger-api/swagger-js/blob/master/docs/MIGRATION_2_X.md. 

I'm trying this without luck - and wanted to ask if this was still the best approach. In the docs on Swagger UI 3.x 'authorizations' is not mentioned in the list of supported parameters (https://github.com/swagger-api/swagger-ui#parameters) although in code is it an accepted configuration option (https://github.com/swagger-api/swagger-ui/blob/41d8089d0e368962272c08375aca29d1dc1924a5/src/core/index.js#L9).

If this is not yet implemented or there is a better way (such as accessing the underlying swagger-js client directly) I'd appreciate any suggestions.

Thanks,

**Comment by webron** [comment]
We have pending work to allow that, but it's not merged yet. Can't give you an ETA for it, but hopefully not too long.

**Comment by shockey** [comment]
Closing; work around this can be tracked in https://github.com/swagger-api/swagger-ui/issues/2915.
